docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md

```diff
@@ -7,7 +7,7 @@ limited: true
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-Aiven for OpenSearch enables you to restore data from external OpenSearch or Elasticsearch snapshots, enabling seamless migration from third-party repositories.
+Aiven for OpenSearch lets you to restore data from external OpenSearch or Elasticsearch snapshots, enabling seamless migration from third-party repositories.
```

Comment:
seamless in our context might means different things, after a couple of bugs we are observing, im not sure if we can say it is going to be seamless

Reply:
removed this word